src/formatting/string.rs

```diff
@@ -226,6 +226,24 @@ fn not_whitespace_except_line_feed(g: &str) -> bool {
 /// FIXME(issue#3281): We must follow UAX#14 algorithm instead of this.
 fn break_string(max_width: usize, trim_end: bool, line_end: &str, input: &[&str]) -> SnippetState {
     let break_at = |index /* grapheme at index is included */| {
+        // Ensure break is not after an escape '\' as it will "escape" the `\` that is added
+        // for concatenating the two parts of the broken line.
```

Comment:
Improve the clarity of the comment
```suggestion
        // Ensure we don't break after an escape character '\' as it will "escape" the line continuation character `\`
        /// that is added to concatenate the two parts of the broken line.
```